testing/generation/command_docs/output/copy_test.go

```diff
@@ -23,16 +23,16 @@ func TestCopy(t *testing.T) {
 		Converts("COPY table_name ( column_name , column_name ) FROM ' filename '"),
 		Unimplemented("COPY table_name ( column_name ) FROM PROGRAM ' command '"),
 		Converts("COPY table_name FROM STDIN"),
-		Unimplemented("COPY table_name FROM PROGRAM ' command ' ( FORMAT format_name )"),
```

Comment:
Lots of changes showing up in this file, but they are mostly changing `format_name` to `text`. Our parser only accepts `BINARY`/`TEXT`/`CSV` for `COPY FROM` statements, so `format_name` would never parse for us.